page.codeberg.petsoi.wordle.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "page.codeberg.petsoi.wordle",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "wordle",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "modules": [
+        {
+            "name": "wordle",
+            "buildsystem": "meson",
+            "build-options": {
+                "append-path": "/usr/lib/sdk/rust-stable/bin"
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://codeberg.org/petsoi/wordle.git",
```

Comment:
Drop touch from recommends https://codeberg.org/petsoi/wordle/src/commit/eca84e77fa01f17d256c7c8e3e92a9664e928e39/data/page.codeberg.petsoi.wordle.metainfo.xml.in.in#L46-L48 you can't put same thing under two relations.